apps/ocr-extraction/app/pdf_ocr.py

```diff
@@ -176,7 +175,8 @@ def response_handler(self, query, blobs, response, r_blobs):
         total_segments = 0
         for uniqueid, url, b in zip(uniqueids, urls, r_blobs):
             if self.has_text_content(b):
-                logger.info("PDF has extractable text, skipping OCR processing")
+                logger.info(
+                    "PDF has extractable text, skipping OCR processing")
```

Comment:
So this means that if a PDF has text, we don't OCR at all?

Reply:
Yes. That's the intent. I'll add a comment.